### ISSUE TITLE
Make numericInput work with complex masks

### ISSUE THREAD

**Issue by NikKosse** [RobinHerbots/Inputmask#963]
I am trying to use your library like so:

``` javascript
 $(this).inputmask("currency", {
                radixPoint: ".",
                groupSeparator: ",",
                digits: 2,
                autoGroup: true,
                removeMaskOnSubmit: false,
                prefix: "$",
                showMaskOnHover: false,
                allowPlus: false,
                allowMinus: false
            });
```

But when I try to add `numericInput: true` in there, it gives me an error saying that there is an error in the code or in my mask definition.  My ultimate goal is to have the input be handled so that to get $15.00 you just have to enter 1500.  From reading [this thread](https://github.com/RobinHerbots/jquery.inputmask/issues/644) it is my understanding that using numericInput the way I want to will not work, as of that thread, close to a year ago.  Is that still the case, and do you know if there is a way to get what I want using your libary?

**Comment by NikKosse** [comment]
Is there a way to accomplish our goal using your plugin?  Not to be a bother, but if you could get back  to me on this it would be great!  To make it clear (hopefully), the ultimate goal is to have the caret be on the far right side of the input box when clicking on said box, and remain there while inputting numbers.  We also want it to be formatted as a dollar amount, with the dollar sign, decimals for the cents, and commas every three digits.

**Comment by RobinHerbots** [comment]
@NikKosse , 

The numericInput options now also works for complexer masks.  

In the numeric alias there are still some issues when using the numericInput when deleting the content.

But that I will fix in #644
